lib/generator._js

```diff
@@ -95,6 +96,18 @@ function ScriptGenerator(repositoryRoot, projectType, projectPath, solutionPath,
     var relativeSolutionPath = path.relative(repositoryRoot, solutionPath);
     log.info('Solution file path: .' + path.sep + relativeSolutionPath);
     this.solutionPath = relativeSolutionPath;
+
+    // find .xproj file, should be under projectPath
+    var files = fs.readdirSync(projectPath);
+    var webProjFiles = files.filter(function(file){
+      return file.endsWith(".xproj");
```

Comment:
Which node version did you test this with? I think `.endsWith` is pretty new to JavaScript, but I'm not sure when node started supporting that. I'd make sure to test with node 4.x as well as 6.x